src/Compilers/CSharp/Portable/Symbols/SpecialTypeExtensions.cs

```diff
@@ -6,27 +6,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal static class SpecialTypeExtensions
     {
-        /// <summary>
-        /// Checks if a type is considered a "built-in integral" by CLR.
-        /// </summary>
-        public static bool IsIntegralType(this SpecialType specialType)
-        {
-            switch (specialType)
-            {
-                case SpecialType.System_Byte:
-                case SpecialType.System_SByte:
-                case SpecialType.System_Int16:
-                case SpecialType.System_UInt16:
-                case SpecialType.System_Int32:
-                case SpecialType.System_UInt32:
-                case SpecialType.System_Int64:
-                case SpecialType.System_UInt64:
-                    return true;
-                default:
-                    return false;
-            }
-        }
```

Comment:
same impl as VB.  moved to common location so C#/VB compilers can reference this, as can IDE.